js/phemex.js

```diff
@@ -1063,31 +1071,32 @@ module.exports = class phemex extends Exchange {
         };
         const duration = this.parseTimeframe (timeframe);
         const now = this.seconds ();
-        const maxLimit = 2000; // maximum limit, we shouldn't sent request of more than it
+        const possibleLimitValues = [ 5, 10, 50, 100, 500, 1000 ];
+        const maxLimit = 1000; // maximum limit, we shouldn't sent request of more than it
         if (limit === undefined) {
             limit = 100; // set default, as exchange doesn't have any defaults and needs something to be set
-        } else {
-            limit = Math.min (limit, maxLimit);
         }
-        if (since !== undefined) {
-            limit = Math.min (limit, maxLimit);
+        limit = Math.min (limit, maxLimit);
```

Comment:
@AlgoQ `fetchOHLCV´is not working for linear USD settled markets, can you check it out pls?
```
p phemex fetchOHLCV "BTC/USD:USD"          
Python v3.10.9
CCXT v2.7.62
phemex.fetchOHLCV(BTC/USD:USD)
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt12/ccxt/examples/py/cli.py", line 232, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/cjg/Git/ccxt12/ccxt/examples/py/cli.py", line 211, in main
    result = await method(*args)
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/phemex.py", line 1066, in fetch_ohlcv
    response = await getattr(self, method)(self.extend(request, params))
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/base/exchange.py", line 1319, in request
    return await self.fetch2(path, api, method, params, headers, body, config, context)
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/base/exchange.py", line 1316, in fetch2
    return await self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/base/exchange.py", line 163, in fetch
    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/phemex.py", line 3507, in handle_errors
    self.throw_exactly_matched_exception(self.exceptions['exact'], errorCode, feedback)
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/base/exchange.py", line 1619, in throw_exactly_matched_exception
    raise exact[string](message)
ccxt.base.errors.BadRequest: phemex {"code":412,"msg":"Missing parameter - to","data":null}
```